src/app/components/modules/Modals.jsx

```diff
@@ -66,6 +85,15 @@ class Modals extends React.Component {
               })
             : [];
 
+        const buySteemPower = e => {
+            if (e && e.preventDefault) e.preventDefault();
+            const new_window = window.open();
+            new_window.opener = null;
+            new_window.location =
+                'https://blocktrades.us/?input_coin_type=eth&output_coin_type=steem&receive_address=' +
```

Comment:
Right now, it points to buy 'Steem' from Blocktrades instead of 'Steem Power'. Looks like we need to use `coin_type=steem_power`.

![screen shot 2018-03-21 at 3 00 34 pm](https://user-images.githubusercontent.com/1121139/37739820-b98c7876-2d18-11e8-8759-cc546049de84.png)

Reply:
Ok thanks.